### PR TITLE
FEAT: Added cart page (without functionality)

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added cart page (Without functionality)
- added route to cart page in "App.js"

Files to check:
- components/cart/Cart.jsx
- components/cart/CartTotal.jsx
- components/cart/CartData.js

### CODE REVIEW

File: src/components/cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                  className="card-img img-horizontal"
+                  src={imageSrc}
+                  alt={name}
+                />
+                {hasOffer && <span className="card-badge">{badgeMessage}</span>}
+                <span className="hide dismiss-btn">X</span>
+                {/* <span className="rated-star rating-horizontal">
+                  {<RatingIcon className="rated rated-star rating-size" />}
+                  {rating}
+                </span> */}
```
Comment (iRohitGaur) [inline src/components/cart/Cart.jsx]: remove commented code

**Review by iRohitGaur — APPROVED** [review]
UI looks great. Nice work! Good to merge 👍🏻

File: src/components/cart/Cart.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              <div className="card-about horizontal-about">
+                <h1 className="title">{name}</h1>
+                <h2 className="sub-title card-sub-title">{description} </h2>
+                <p className="card-para card-horizontal-para">
+                  <div className="card-hori-div1">
+                    ₹{discountPrice}
+                    {hasDiscount && (
+                      <span className="striken-text"> ₹{originalPrice}</span>
+                    )}
+                    <br />
```
Comment (amartya-rs) [inline src/components/cart/Cart.jsx:53]: Instead of br tags, we should use margin I guess

**Review by amartya-rs — APPROVED** [review]
Good to merge with suggested changes.

**Review by sudipkundu999 — APPROVED** [review]
Just a UI suggestion: The product card looks small and cartTotal looks a little big. If they could be on similar lines then it would look better.
Attaching an image for your reference:
Screenshot (88)

**Comment by zoyatanweer** [comment]
> Just a UI suggestion: The product card looks small and cartTotal looks a little big. If they could be on similar lines then it would look better. Attaching an image for your reference: Screenshot (88)

Thanks for this suggestion Sudip! Will surely work on it.

**Comment by zoyatanweer** [comment]
Thankyou all for your reviews, will make the suggested changes.

**Pull request merged**